grammars/css.cson

```diff
@@ -494,17 +494,15 @@
         ]
       }
       {
-        'begin': '(:)\\s*'
+        'begin': ':'
```

Comment:
I suppose you could argue that the no spaces version is cleaner for background highlighting.  This is also how language-less and sass do it.